packages/framework/src/Form/Admin/Transport/TransportFormType.php

```diff
@@ -47,11 +52,22 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         new Constraints\Length(['max' => 255, 'maxMessage' => 'Name cannot be longer than {{ limit }} characters']),
                     ],
                 ],
+                'label' => t('Name'),
             ])
             ->add('domains', DomainsType::class, [
                 'required' => false,
+                'label' => t('Display on'),
             ])
-            ->add('hidden', YesNoType::class, ['required' => false])
+            ->add('hidden', YesNoType::class, [
+                'required' => false,
+                'label' => t('Hidden'),
+            ]);
```

Comment:
Im sorry, but first, it does not make any sense to me why. Ans second of all, there is not single usage of this and i would make it incosistent or i would spend hours on remaking it in whole project, standards does not input this at all.

Reply:
Again, this is how it is done throughout whole project. If i will change it here, than it will be uncosistent with other parts of the application, if it really bothers you, create a user story where we can aggree that this is how we want to indent fluents, then add it into standards check and than i will change it, but now it is just your argument vs my argument and i think that this indention is correct.

Reply:
And other thing is that even if i would like to do it, right now standards are set to not let this happen.